docs/source/beersheba.rst

```diff
@@ -251,6 +254,60 @@ Deconvolution process finishes when any of the stopping criteria occurs:
 RL deconvolution is implemented in Beersheba using the Richardson-Lucy function from Python’s scikit-image library. Additional
 details about how the RL deconvolution works and the way the PSFs are implemented can be found in the RL publication [#]_.
 
+.. _Satellite-Killer:
+
+Satellite Killer
+:::::::::::::::::
+
+A common issue within the deconvolution process is the creation of 'satellite tracks'.
+These are small artificial energy deposits that exceed the later applied cleaning cut, generated
+due to energy that is unable to be deconvolved back to it's initial point and instead coalescing
+around the main track (hence satellites). This can be seen in the plot below on the left.
+
+**Satellite killer** is an algorithm introduced to remove these satellites *during* the deconvolution
+process, allowing for a resulting track that contains no artificial hits as seen on plot below on the right. 
+It is fine-tunable, allowing the end user to change the strength of the implementation easily.
+
+.. image:: images/beersheba/satellites.png
+  :width: 100%
+
+These parameters are stored within the dictionary ``satellite_params``, and described below:
+
+.. |sk| raw:: html
+
+     <sk>
+
+.. list-table::
+   :widths: 50 40 120
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type (Option)**
+     - **Description**
+
+   * - ``satellite_start_iter``
+     - ``int``
+     - Iteration no. when satellite killer starts being applied.
+
+   * - ``satellite_max_size``
+     - ``int``
+     - Maximum size of a satellite deposit, above which they are considered 'real'.
+
+   * - ``e_cut``
+     - ``float``
+     - Cut in absolute/relative value to the provided deconvolution output for satellite discrimination.
+
+   * - ``cut_type``
+     - ``str`` ('**abs**', '**rel**')
```

Comment:
> Should I alter those to also show the correct types? 

Yes, please, let's take the chance to update them. Even though we need to do that everywhere, let's focus on this file for now. Not sure what the correct display should be. Definitely not quotes, as it is misleading. Perhaps `HitEnergy` (**E**, **Ec**).

> I'm assuming these were labelled as strings to help the users, who will input the values as "strings" without having to worry about the custom types of CutType and Hit_Energy.

That was before. Now users don't input strings, but enum members (aka symbols)

Reply:
Got it, I'll alter these now in beersheba. Perhaps (so we dont forget) it would be worth opening an issue to alter these in all docs once the other city-based PRs have been merged?